src/layout/organize/JourneysLayout.tsx

```diff
@@ -11,6 +11,24 @@ const JourneysLayout: React.FunctionComponent = ({ children }) => {
     <TabbedLayout
       baseHref={`/organize/${orgId}/journeys`}
       defaultTab="/overview"
+      ellipsisMenuItems={[
+        {
+          label: intl.formatMessage({
+            id: 'layout.organize.journeys.menu.downloadCsv',
+          }),
+          onSelect: () => {
+            location.href = `/api/journeyInstances/download?orgId=${orgId}`;
+          },
+        },
+        {
+          label: intl.formatMessage({
+            id: 'layout.organize.journeys.menu.downloadXlsx',
+          }),
+          onSelect: () => {
+            location.href = `/api/journeyInstances/download?orgId=${orgId}&format=xlsx`;
```

Comment:
I'm honestly not sure if there is any difference between doing this and using the router. I did this because I wanted the browser to trigger a download, which it does (it does not navigate, unless there is an error). I didn't use the router because this is completely outside the scope of what the router brings to the table, but it probably would work (and it would just do the same thing internally).

I can give it a try if you want?